src/snyk/common/configuration/configuration.ts

```diff
@@ -195,6 +189,7 @@ export class Configuration implements IConfiguration {
   get authHost(): string {
     if (this.customEndpoint) {
       const url = new URL(this.customEndpoint);
+      url.host = url.host.replace('api', 'app');
```

Comment:
Let's also remove the `/api` path component, as it's not valid for `api.xxx.snyk.io`